src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -66,23 +66,23 @@
       </v-row>
 
       <v-row
-        class="mt-8"
+        class="mt-6"
         no-gutters
       >
         <v-col
           cols="12"
           sm="3"
         >
-          <label>Business Information</label>
+          <label>Extraprovincial Registration Information</label>
```

Comment:
This is this component:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/11c4b6d9-3163-460a-99d2-2ecbe5dcaeb8)